kivymd/uix/templates/scalewidget/scalewidget.py

```diff
@@ -15,7 +15,7 @@
 
 .. code-block:: python
 
-    
+
```

Comment:
There must be only one newline

Reply:
sorry, forgot that pre-commit does not check the docstrings